src/twisted/python/htmlizer.py

```diff
@@ -60,35 +71,58 @@ def __init__(self, writer):
         reflect.accumulateClassList(self.__class__, "noSpan", noSpan)
         self.noSpan = noSpan
 
+
     def write(self, token, type=None):
-        token = cgi.escape(token)
+        if _PY3 and isinstance(token, bytes):
```

Comment:
Is there any reason this can't be an unconditional `.decode("utf-8")` for bytes, even on py2?